buildSrc/subprojects/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle

```diff
@@ -176,24 +190,28 @@ performanceTest.registerTestProject("nativeMonolithicOverlapping", MonolithicNat
     // number of source files to generate in each project of each type
     // this generates 200 C, 200 C++, 200 headers and 400 unused C/C++ files
     sourceFiles = 200
+    daemonMemory = '2500m'
```

Comment:
Nit: shall we use multiples of 1024, or something like `2.5G`?

Reply:
`2.5G` is not a valid `Xmx` option. I switched to `2560`